### DISCUSSION TITLE
HTML "snippet" injection support by GitHub Pages?

### DISCUSSION THREAD

**Discussion by chalin** [community/community#34823] [Other Feature Feedback, Questions, & Ideas]
Does GitHub Pages support an equivalent of Netlify's [Snippet injection](https://docs.netlify.com/site-deploys/post-processing/snippet-injection/), for injecting HTML (such as analytics scripts) into all served pages? We have multiple repos being served from https://argoproj.github.io, and it would be easier to add analytics via injection than to configure it separately for each repo. Thanks.

**Comment by yoannchaudet** [comment]
Hello,

We do not support any post processing feature like this one today.
